src/ripple/app/tx/Transaction.h

```diff
@@ -77,15 +76,20 @@ class Transaction
         boost::optional<std::uint64_t> const& ledgerSeq,
         boost::optional<std::string> const& status,
         Blob const& rawTxn,
-        Validate validate,
+        Application& app);
+
+    static
+    Transaction::pointer
+    transactionFromSQLValidated (
+        boost::optional<std::uint64_t> const& ledgerSeq,
```

Comment:
Actually I think `ledgerSeq` should be `boost::optional<LedgerIndex> const& ledgerSeq`, and note that `LedgerIndex` is `std::uint32_t`.